vulnerabilities/importers/debian_oval.py

```diff
@@ -8,38 +8,29 @@
 #
 
 
-import asyncio
 import xml.etree.ElementTree as ET
 
 import requests
 
 from vulnerabilities.importer import OvalImporter
-from vulnerabilities.package_managers import DebianVersionAPI
-from vulnerabilities.utils import create_etag
 
 
 class DebianOvalImporter(OvalImporter):
+    spdx_license_expression = "LicenseRef-scancode-unknown"
```

Comment:
This should be:
```suggestion
    spdx_license_expression = "LicenseRef-scancode-other-permissive"
```
and we should have the exact the same value for https://github.com/nexB/vulnerablecode/blob/ae609a857c747c45c8391b9b0fd3c54b0b30d921/vulnerabilities/importers/debian.py#L42 ...
The license URL and MIT does not apply here IMHO and we should not have a license URL

And the notice in debian should also exist here: https://github.com/nexB/vulnerablecode/blob/ae609a857c747c45c8391b9b0fd3c54b0b30d921/vulnerabilities/importers/debian.py#L44